Mohammad-Husam/coding_challenge
# Feature/create model class to decode json file

**Background**

- Adding data layer


**Modifications**

- Converting JSON file to dart file using: https://javiercbk.github.io/json_to_dart/
- Adding JSON serializer (to convert the data from JSON to JSON).
- Testing the data on the console.

**Step to test**

- clone the project into your device.
- connect physical or emulator device.
- press on the red container in the center of screen.
- observe the result on the console.
